crates/forge/src/scarb.rs

```diff
@@ -25,19 +25,27 @@ impl PackageConfig for ForgeConfig {
     }
 }
 
-pub fn build_contracts_with_scarb(filter: PackagesFilter) -> Result<()> {
+pub fn build_contracts_with_scarb(
+    filter: PackagesFilter,
+    features_spec: FeaturesSpec,
```

Comment:
```suggestion
    features: FeaturesSpec,
```
Inconsistent naming.